src/main/java/org/apache/beam/sdk/io/gcp/healthcare/HealthcareApiClient.java

```diff
@@ -13,15 +13,8 @@
 // limitations under the License.
 package org.apache.beam.sdk.io.gcp.healthcare;
 
-import com.google.api.services.healthcare.v1beta1.model.Empty;
-import com.google.api.services.healthcare.v1beta1.model.FhirStore;
-import com.google.api.services.healthcare.v1beta1.model.Hl7V2Store;
-import com.google.api.services.healthcare.v1beta1.model.HttpBody;
-import com.google.api.services.healthcare.v1beta1.model.IngestMessageResponse;
-import com.google.api.services.healthcare.v1beta1.model.ListMessagesResponse;
-import com.google.api.services.healthcare.v1beta1.model.Message;
-import com.google.api.services.healthcare.v1beta1.model.Operation;
-import com.google.api.services.healthcare.v1beta1.model.ParserConfig;
+import com.google.api.services.healthcare.v1beta1.model.*;
```

Comment:
I would leave all the imports individually defined to allow other contributors a better idea of what is being used.